src/test/java/bio/terra/aws/resource/discovery/EnvironmentTest.java

```diff
@@ -69,10 +70,22 @@ public void inequality() {
     String string = "string";
     Assertions.assertNotEquals((Object) environment, (Object) string);
 
+    // Different Notebook Role Arn
```

Comment:
This is different instance profile name, right?

Reply:
Bad copy-pasting ... thanks!